setup.py

```diff
@@ -58,7 +58,7 @@ def readme():
               'dill>=0.3.0, <0.4.0',
               'transformers>=4.7.0, <5.0.0',
               'tqdm>=4.28.1, <5.0.0',
-              'pydantic <2.0.0',
+              'pydantic<3.0.0',
```

Comment:
I dont think we even need to specify pydantic here as it is a sub dependency of ray and not related to alibi

Reply:
✅